k6ReportPortal.js

```diff
@@ -37,92 +40,155 @@ export function finishLaunch(launchId,reporterOptions) {
  * @returns 
  */
 function getHeader(token) {
-    var header = {
+    return {
         headers: {
-            "Content-Type": "application/json",
+            'Content-Type': 'application/json',
             authorization: `Bearer ${token}`,
         },
     };
-    return header;
-}
-
+};
 
 /**
  * 
  */
 export default class RpClient {
-
-    constructor(launchId,reporterOptions){
+    constructor(launchId, reporterOptions) {
         this.launchId = launchId
         this.reportPortalUri = `${reporterOptions.endpoint}/${reporterOptions.project}`
         this.token = reporterOptions.token
+    }
 
+    /**
+     * This will return the value of a log batch with a new log entry added.
+     * 
+     * @param {Array} jsonObject
+     * @param {string} message
+     * @param {string} level 
+     * @returns 
+     */
+    addLogToBatch(jsonObject, message, id, level = 'error') {
+        let newObject = jsonObject;
+        newObject.push({
+            message: message,
+            time: Date.now(),
+            launchUuid: this.launchId,
+            itemUuid: id,
+            level: level
+        });
+        return newObject;
+    }
 
+    /**
+     * This will upload the log batch to the launch.
+     * 
+     * @param {Array} jsonBody
+     */
+    saveLogBatch(jsonBody) {
```

Comment:
Can we change the name as writeLogBatch? Like writeLog is for single log and writeLogBatch is for multiple batches of log